openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -0,0 +1,112 @@
+from rest_framework.views import APIView
+from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.auth.models import User
+from rest_framework.response import Response
+from rest_framework import status
+
+from student.models import UserProfile
+from openedx.core.djangoapps.user_api.accounts.serializers import AccountLegacyProfileSerializer, AccountUserSerializer
+from openedx.core.lib.api.permissions import IsUserInUrlOrStaff
+
+from rest_framework.authentication import OAuth2Authentication, SessionAuthentication
+from rest_framework import permissions
+from rest_framework import status
+
+
+class AccountView(APIView):
+    """
+        **Use Cases**
+
+            Get the user's account information.
+
+        **Example Requests**:
+
+            GET /api/user/v0/accounts/{username}/
+
+        **Response Values**
+
+            * username: The username associated with the account (not editable).
+
+            * name: The full name of the user (not editable through this API).
+
+            * email: The email for the user (not editable through this API).
+
+            * date_joined: The date this account was created (not editable).
```

Comment:
I don't think that users of an API would assume that it was implemented in Python. Maybe there is an implied date format that all REST APIs use, but I'm not sure if that's true:

http://stackoverflow.com/questions/10286204/the-right-json-date-format